user_guide_src/source/changelogs/v4.2.8.rst

```diff
@@ -12,7 +12,8 @@ Release Date: Unreleased
 BREAKING
 ********
 
-none.
+- ``CodeIgniter\Database\BasePreparedQuery`` class return now a bool value for write-type queries instead of the ``Result`` class object.
```

Comment:
```suggestion
- ``CodeIgniter\Database\BasePreparedQuery`` class returns now a bool value for write-type queries instead of the ``Result`` class object.
```